ilastik/app.py

```diff
@@ -256,6 +260,27 @@ def _update_hbp_mode(parsed_args):
         ilastik_config.set("ilastik", "hbp", "true")
 
 
+def _update_tiktorch(parsed_args, root_path: str):
+    """enable tiktorch local workflow"""
+    tiktorch_executable: Optional[Path] = None
+    if parsed_args.tiktorch_executable:
+        tiktorch_executable = Path(parsed_args.tiktorch_executable)
+    else:
+        # Maybe tiktorch is bundled
+        root_path = Path(root_path)
+        tiktorch_script = "run_tiktorch.sh" if platform.system() != "Windows" else "run_tiktorch.bat"
+        bundled_tiktorch_executable = root_path / "tiktorch" / tiktorch_script
+
+        if bundled_tiktorch_executable.exists():
+            tiktorch_executable = bundled_tiktorch_executable
+
+    if tiktorch_executable and tiktorch_executable.exists():
+        tiktorch_msg = "Using tiktorch executable: %s" % tiktorch_executable
+        print(tiktorch_msg)
```

Comment:
Nope, this is intentional because logger may be uninitialized and we want to have this output. Same with the message "Starting ilastik from ..." in this file